backend/clubs/management/commands/import_paideia_events.py

```diff
@@ -0,0 +1,69 @@
+import json
+import re
+import urllib.request
+from datetime import datetime
+from html import unescape
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from django.utils.html import strip_tags
+
+from clubs.models import Event
+
+
+class Command(BaseCommand):
+    help = (
+        "Imports events from SNF Paideia's site "
+        "into the Events model at a set frequency"
+    )
+
+    def handle(self, *args, **kwargs):
+        url = "https://snfpaideia.upenn.edu/wp-json/wp/v2/event"
+
+        with (urllib.request.urlopen(url)) as data:
+            parsed_json = json.loads(data.read().decode())
+            for event in parsed_json:
+
+                # prevent duplicates
+                existing = Event.objects.filter(code=event["slug"]).first()
+                if existing:
+                    existing.delete()
+
+                ev = Event()
+                # ev.club = Is there a club for this? What even is SNF Paideia?
+                ev.code = event["slug"]
+                ev.name = unescape(event["title"]["rendered"])
+                ev.description = (
```

Comment:
I think `event["excerpt"]` might be a better description/introduction. What do you think?